README.md

```diff
@@ -121,32 +132,110 @@ import { Live2DModel } from 'pixi-live2d-display';
 // reference window.PIXI.Ticker to automatically update Live2D models
 window.PIXI = PIXI;
 
+var model_proxy; //make a global scale handler to use later
```

Comment:
![image](https://github.com/guansss/pixi-live2d-display/assets/34002411/afb0fc20-d574-4c76-afd6-ea9fa4b9ff49)
well you used a scoped code to generate model as local variable, So, I assigned it globally to use later 😅

Reply:
I mean, generally, examples are supposed to run individually, so it's unnecessary to make example B to reference a variable in example A.
We can simply use `model` in the follow-up examples (just like many examples in the docs site), and users will know how to create `model` because it's already shown in the first example.

Also, I just realized this example should go to the docs (`./docs/docs/`), since the readme is just a place to show very simple and basic examples.
The docs are using in an extended Markdown syntax and requires setting up some Python environment if you want to locally preview them. Later I'll add these instructions to `DEVELOPMENT.md`.

Reply:
It'd be really helpful if you add them to the docs (never did docs before), I've written things quite clear and understandable (updating a new branch from your latest comments to fix them, only a few, left, just busy to complete and submit)

Reply:
Okay I will do it when you've finished them.

Reply:
Yay, thanks 😸